doc/guidelines/frontend-ui-usability-guide.md

```diff
@@ -0,0 +1,166 @@
+- When **active**, i.e. currently activated/enabled, they should get a small **highlighting** through a visually stronger and more colourful appearance by using the design's primary/secondary colours.
+- When **disabled**, i.e. currently not clickable, they should be **greyed-out**, either via decreasing its opacity or choosing light, faint colours.  
+  The mouse **cursor** should be styled with the **_not-allowed_** css property.
+- A **tooltip** on hovering should be displayed for every icon in PermaplanT's toolbox.
+  The tooltip should contain the icon's **label** (as concise as possible) and, if existing, the assigned **shortcut**.
+
+### Highlighting
+
+Highlighting techniques:
+
+- Font weight:
+
+  - **Bold** text adds tolerable noise to the design and enables distinct highlighting in both short and long texts.
+    In general, it is the preferred highlighting technique to use.
+  - _Italic_ text adds minimal noise to the design, but lacks in recognizability.
+  - <u>Underlining</u> adds most noise and compromises a text's legibility.
+    It should hardly ever be used.
+
+- Colours: primary and secondary colours of the design's colour palette
+
+- Images: users generally remember images better than words
+
+- Avoid different fonts for highlighting purposes.
+
+### Wording
+
+Messages shown to the user should strive to fulfill following criteria:
+
+- **concise**: the more text the less likely it will be read by the user
+- **clear**: straight to the message's essence
+- **understandable** for technical laymen: no error codes or technical terms
+- **no exclamation** marks: interpreted as commanding
+- **no uppercase** words: comes across as shouting
+- **detailed** information should be hidden behind a **read more** link or a **collapsed** section
+- **Headlines**, **sub-headlines** and **labels** should be **capitalized**
+- **Minimize hyphenation** of words
+
+### Error Messages
+
+In general, error messages often convey the impression that an application's stability is weak which, in consequence, leads to a **declined user experience**.
+Error messages should **only** be used **if** the **user**, without that information, **would be badly surprised** by the result of an action.
+
+Error messages should fulfill following criteria:
+
+- **polite** and **neutral**: do not directly or indirectly blame the user and stay away from jokes
+- **brief**
+- **specific** to the problem
+- contain **reason for the problem**
+- **solution-oriented**: ideally, a solution to the occurred problem is offered
+- no **privacy-violating** information: e.g. informing the user upon a failed login that the entered email does not exist in the system - this could additionally also encourage people to scan our database for emails
+
+### Search Fields
+
+To prevent _Chromium_ and _WebKit_ based browsers from adding their own cross icon and functionality into search input fields, **do not declare** input fields with **type _search_**.  
```

Comment:
Start with important+general information, rationale+details later.

```suggestion
Do not declare input fields with type `search`.
_Rationale:_  To prevent _Chromium_ and _WebKit_ based browsers from adding their own cross icon and functionality into search input fields.
```